world/stat_checks/check_maker.py

```diff
@@ -20,9 +20,15 @@ def check_rolls_tied(roll1, roll2, tie_value=TIE_THRESHOLD):
 
 class SimpleRoll:
     def __init__(
-        self, character=None, stat=None, skill=None, rating: DifficultyRating = None
+        self,
+        character=None,
+        stat=None,
+        skill=None,
+        rating: DifficultyRating = None,
+        receivers: list = None,
     ):
         self.character = character
+        self.receivers = receivers
```

Comment:
For receiving an iterable argument that can be None it's safer to set it to be an empty list/iterable so that it is still iterable and won't raise type errors if you try to iterate on it. So I'd prefer:
`self.receivers = receivers or []` for example so that it would be an empty list rather than None when not specified.